maas/client/viscera/events.py

```diff
@@ -93,7 +95,8 @@ class EventsType(ObjectType):
             level: typing.Union[Level, int, str]=None,
             before: int=None,
             after: int=None,
-            limit: int=None):
+            limit: int=None,
+            owner: str=None):
```

Comment:
It would be nice to be able to pass either a string or a `User`.

`owner: Union[User, str]=None`